inc/saiport.h

```diff
@@ -2108,6 +2120,233 @@ typedef sai_status_t (*sai_clear_port_pool_stats_fn)(
         _In_ uint32_t number_of_counters,
         _In_ const sai_stat_id_t *counter_ids);
 
+/**
+ * @brief List of Port Serdes attributes
+ */
+typedef enum _sai_port_serdes_attr_t
+{
+    /**
+     * @brief Start of attributes
+     */
+    SAI_PORT_SERDES_ATTR_START,
+
+    /**
+     * @brief Port ID
+     *
+     * @type sai_object_id_t
+     * @flags MANDATORY_ON_CREATE | CREATE_ONLY
+     * @objects SAI_OBJECT_TYPE_PORT
+     */
+    SAI_PORT_SERDES_ATTR_PORD_ID = SAI_PORT_SERDES_ATTR_START,
```

Comment:
Typo : PORD ==> PORT

Can we not reuse a serdes object (basically serdes settings) across similar ports ? Why would we want to tie to a specific port ?

Reply:
Done